src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const discount = calculateDiscount(days);
+  return days * 40 - discount;
```

Comment:
The calculation of the final rental cost should directly include the discount in its calculation rather than subtracting it later. Consider revising this line to ensure the discount is applied correctly.